node/bindnode/infer_test.go

```diff
@@ -89,6 +89,34 @@ var prototypeTests = []struct {
 			"linkImpl":    {"/": "bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi"}
 		}`,
 	},
+	{
+		name: "Enums",
+		schemaSrc: `type Root struct {
+				enumAsString       EnumAsString
+				enumAsStringCustom EnumAsString
+				enumAsInt          EnumAsInt
+			}
+			type EnumAsString enum {
+				| Nope ("No")
+				| Yep  ("Yes")
+				| Maybe
+			}
+			type EnumAsInt enum {
+				| Nope  ("10")
```

Comment:
i don't see where these will get parsed to ints. is that already happening?

Reply:
You reviewed that right before the holidays :) https://github.com/ipld/go-ipld-prime/pull/317